app/models/card.rb

```diff
@@ -80,7 +80,7 @@ def self.new_with_new_deck(card_params, deck_params, user)
 
   def not_equal_fields
     if clear(original_text) == clear(translated_text)
-      errors.add(:translated_text, "can't be the same as Original text")
+      errors.add(:translated_text, I18n.t('cards.cant_be_equal'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.